Multisig/UI/Safe Management/Load Safe/EnterSafeAddress/EnterSafeAddressViewModel.swift

```diff
@@ -80,7 +80,11 @@ class EnterSafeAddressViewModel: ObservableObject {
             }
             .sink(receiveCompletion: { completion in
                 if case .failure(let error) = completion {
-                    self.setError(error.localizedDescription)
+                    if case HTTPClient.Error.entityNotFound(_, _, _) = error {
+                        self.setError("We could not find a safe with this address.")
+                    } else {
```

Comment:
As far as I tried with the [invalid safe address "123"](https://safe-transaction.staging.gnosisdev.com/api/v1/safes/123/), the 422 would mean here that the user address is not valid, so I would add it here with a human-readable message instead/or in addition to the "Unprocessable entity"

Reply:
I think we have client validation of valid addresses; can you tell me how you managed to trigger 422 error?

Reply:
yes, that's right. I triggered by following the URL in the comment above.